src/AutoWrapper.Test/Helper/JsonSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+using System.Collections.Generic;
+
+namespace AutoWrapper.Test.Helper
+{
+    internal class CamelCaseContractResolverJsonSettings
+    {
+        public JsonSerializerSettings GetJSONSettings(bool ignoreNull, ReferenceLoopHandling referenceLoopHandling = ReferenceLoopHandling.Ignore, bool useCamelCaseNaming = true)
+        {
+            return new JsonSerializerSettings
+            {
+                ContractResolver = useCamelCaseNaming ? new CamelCasePropertyNamesContractResolver() : new DefaultContractResolver(),
+                Converters = new List<JsonConverter> { new StringEnumConverter() },
+                NullValueHandling = ignoreNull ? NullValueHandling.Ignore : NullValueHandling.Include,
+                ReferenceLoopHandling = referenceLoopHandling
+            };
+        }
+    }
+
+    internal class CustomContractResolverJsonSettings<T>
+    {
+        public (JsonSerializerSettings Settings, Dictionary<string, string> Mappings) GetJSONSettings(bool ignoreNull, ReferenceLoopHandling referenceLoopHandling = ReferenceLoopHandling.Ignore, bool useCamelCaseNaming = true)
+        {
+            var resolver = new CustomContractResolver<T>(useCamelCaseNaming);
+            var propMappings = resolver._propertyMappings;
+
+            var settings = new JsonSerializerSettings
+            {
+                ContractResolver = resolver,
+                NullValueHandling = ignoreNull ? NullValueHandling.Ignore : NullValueHandling.Include,
+                ReferenceLoopHandling = referenceLoopHandling
+            };
+
+            return (settings, propMappings);
+
+        }
+    }
```

Comment:
And this one? https://github.com/proudmonkey/AutoWrapper/blob/master/src/AutoWrapper/Helpers/JsonSettings.cs